frontend_app_workspace_advanced/src/container/WorkspaceAdvanced.jsx

```diff
@@ -478,7 +521,39 @@ export class WorkspaceAdvanced extends React.Component {
           default: this.sendGlobalFlashMessage(props.t('Error while adding the member to the space'), 'warning')
         }
         break
-      default: this.sendGlobalFlashMessage(props.t('Error while adding the member to the space', 'warning'))
+      default: this.sendGlobalFlashMessage(props.t('Error while adding the member to the space'), 'warning')
+    }
+  }
+
+  handleClickAcceptRequest = async userId => {
+    const { props, state } = this
+    const fetchPutSubscriptionAccept = await putSubscriptionAccept(
```

Comment:
you need to call handleFetchResult on every async requests

Reply:
Updated